changelog/@unreleased/pr-6950.v2.yml

```diff
@@ -0,0 +1,7 @@
+type: fix
+fix:
+  description: We now start antithesis fuzzer before picking workflow and make our
+    logging synchronous for that project (to make sure it's written to a file and
+    can be picked up before actually running the code that specifies workflows)
```

Comment:
nit: The main thing is that the container now actually selects a random workflow right? (Because the fuzzer picks which workflow is run.) We should probably say that